### PR TITLE
`Development`: Add JavaDocs to UtilServices and Factories [13]

### PR DESCRIPTION
**Pull request by DominikRemo**
#### General
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://ls1intum.github.io/Artemis/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://ls1intum.github.io/Artemis/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context

Improve the readability of UtilServices and Factories by adding JavaDoc to the helper methods

### Description

Added documentation to following files:
- [x] tutorialgroups/TutorialGroupFactory.java
- [x] tutorialgroups/TutorialGroupUtilService.java

#### Code Review
- [ ] Code Review 1
- [ ] Code Review 2

### CODE REVIEW

File: src/test/java/de/tum/in/www1/artemis/tutorialgroups/TutorialGroupUtilService.java [hunk 1]
```diff
@@ -62,13 +62,25 @@ public class TutorialGroupUtilService {
     @Autowired
     private TutorialGroupRegistrationRepository tutorialGroupRegistrationRepository;
 
+    /**
+     * Creates and saves a Course for TutorialGroup tests
```
Comment (chrisknedl) [inline src/test/java/de/tum/in/www1/artemis/tutorialgroups/TutorialGroupUtilService.java:66]:
```suggestion
     * Creates and saves a Course for TutorialGroup tests.
```

File: src/test/java/de/tum/in/www1/artemis/tutorialgroups/TutorialGroupUtilService.java [hunk 2]
```diff
@@ -83,6 +95,14 @@ public TutorialGroupSession createIndividualTutorialGroupSession(Long tutorialGr
         return tutorialGroupSession;
     }
 
+    /**
+     * Creates and saves a TutorialGroupFreePeriod for the TutorialGroupsConfiguration with the given ID.
+     *
+     * @param tutorialGroupsConfigurationId The ID of the TutorialGroupsConfiguration
+     * @param date                          The date of the TutorialGroupFreePeriod
+     * @param reason                        The reason of the TutorialGroupFreePeriod
```
Comment (chrisknedl) [inline src/test/java/de/tum/in/www1/artemis/tutorialgroups/TutorialGroupUtilService.java:103]:
```suggestion
     * @param reason                        The reason for the TutorialGroupFreePeriod
```

**Review by chrisknedl — COMMENTED** [review]
Looks good already, only two minor suggestions.

**Review by egekurt123 — APPROVED** [review]
text lgtm

**Review by chrisknedl — APPROVED** [review]
Looks good 👍

**Pull request merged**